packages/ai/src/models.generated.ts

```diff
@@ -876,6 +876,23 @@ export const MODELS = {
 			contextWindow: 400000,
 			maxTokens: 128000,
 		} satisfies Model<"openai-responses">,
+		"gpt-5.1-codex-max": {
```

Comment:
Was this manually added? ai/scripts/generate-models.ts generates this file, so if we want to manually add this, it needs to go there.